lively.ide/studio/controls/body.cp.js

```diff
@@ -85,6 +85,8 @@ export class BodyControlModel extends PropertySectionModel {
    */
   ensureDynamicControls () {
     this.dynamicControls.forEach(m => m.remove());
+    // FIXME: adjusting the resize policies should automatically cause an override
+    this.view.layout = this.view.layout; // ensure the layout is overridden
```

Comment:
This is really a layout problem. Currently in place modification of the layout does not really refresh reliably. We have to trigger the setter, since this is the only way it is going to ensure the layout is *applied*. We can go ahead and fix this, but I would argue that is a separate issue.